File: C++/Q2.cpp

#include "iostream"
using namespace std;

int main(void)
{

    int num; 

    cout << "Enter an integer: ";

    cin >> num;

    cout << "Reversed number = ";

    while(num != 0)   
    {
        cout << num % 10;

        num /= 10;
    }

    cout << endl;

    return 0;
    
}

File: Python/Q2.py

Num = int(input("Please enter any number: "))
Sum = 0
Avg = 0

for i in range(Num):  
    Sum += i+1

Avg = float(Sum) / Num

print("The Sum of Natural Numbers from 1 to {} = {}".format(Num, Sum))

print("The Average of natural numbers from 1 to {} = {}".format(Num, Avg))